iot/api-client/manager/src/test/java/com/example/cloud/iot/examples/ManagerIT.java

```diff
@@ -115,6 +115,16 @@ public void clearTestRegistries() throws Exception {
             DeviceRegistryExample.clearRegistry(CLOUD_REGION, PROJECT_ID, registryId);
           }
         }
+        // Also remove the current test registry
+        try {
+          DeviceRegistryExample.clearRegistry(CLOUD_REGION, PROJECT_ID, REGISTRY_ID);
```

Comment:
What if we run two of these at once?

Reply:
They should have different registry IDs, this is more to protect against a failure in a previous test having a resource conflict in all subsequent tests.

Reply:
It may make sense to use part of a UUID4 in the registry name though to help prevent that but for now the name is calculated as:

```java
private static final String REGISTRY_ID = "java-reg" + (System.currentTimeMillis() / 100L);
```

So that stale registries can be pruned in this step without deleting registries currently being tested (there's a limit of 100 registries per project).